msal/acquire_token_auth_code_parameters.go

```diff
@@ -30,6 +31,11 @@ func createAcquireTokenAuthCodeParameters(scopes []string) *acquireTokenAuthCode
 
 func (p *acquireTokenAuthCodeParameters) augmentAuthenticationParameters(authParams *msalbase.AuthParametersInternal) {
 	p.commonParameters.augmentAuthenticationParameters(authParams)
-	authParams.Redirecturi = "https://login.microsoftonline.com/common/oauth2/nativeclient"
+	if p.RedirectURI != "" {
+		authParams.Redirecturi = p.RedirectURI
+	}
+	if authParams.Redirecturi == "" { // set it to default if it is still not set
+		authParams.Redirecturi = "https://login.microsoftonline.com/common/oauth2/nativeclient"
```

Comment:
I propose making this a parameter so it's clear the caller must supply a value.  What do you think?

Reply:
Oh yes good point. So instead of going in the options struct, it should be in the parameters for both [public client auth code](https://github.com/AzureAD/microsoft-authentication-library-for-go/blob/dev/msal/public_client_application.go#L101) and [confidential client auth code](https://github.com/AzureAD/microsoft-authentication-library-for-go/blob/dev/msal/confidential_client_application.go#L100)

Reply:
If I make RedirectURI required/argument, I need to figure out what to in tests like these https://github.com/AzureAD/microsoft-authentication-library-for-go/blob/dev/msal/public_client_application_test.go#L80

I will work on it, but it will take a bit of time. I don't yet know how those mocks work. Thanks